Dockerfile

```diff
@@ -56,7 +56,8 @@ RUN apt update         && \
         g++-4.8               \
         xz-utils              \
         build-essential       \
-        bash-completion       && \
+        bash-completion       \
+        libssl-dev         && \
```

Comment:
I prefer to move this package upper so that we can focus and minimize the change/diff.